usr/src/uts/common/fs/zfs/arc.c

```diff
@@ -3069,6 +3073,15 @@ arc_evict_state(arc_state_t *state, uint64_t spa, int64_t bytes,
 	num_sublists = multilist_get_num_sublists(ml);
 
 	/*
+	* Do the job for each type of headers (see b_lsize note below)
+	*/
+	for (int l = 0; l < 3 && (total_evicted < bytes || bytes == ARC_EVICT_ALL); l++) {
```

Comment:
`l` is easily misread for `1`, a possible value of `l`.  How about a different variable name?